src/analyze/analyze-verify.c

```diff
@@ -283,5 +314,28 @@ int verify_units(char **filenames, UnitFileScope scope, bool check_man, bool run
                         r = k;
         }
 
+        if (set_isempty(s) || r != 0)
+                return r;
+        else {
```

Comment:
This block can by dedented by one level, since we have `return` in the preceding line.